src/tests/contract_test/wallet_test.cljs

```diff
@@ -103,3 +106,90 @@
               ens-address (contract-utils/call-rpc "ens_addressOf" chain-id input)
               response    (contract-utils/call-rpc "wallet_getAddressDetails" chain-id ens-address)]
         (assert-address-details response)))))
+
+
+(defn assert-suggested-routes
+  [result]
+  (is (vector? (:Best result)))
+  (is (= "Optimism" (get-in result [:Best 0 :From :chainName])))
+  (is (= "0xa" (get-in result [:Best 0 :AmountIn])))
+  (is (= "Transfer" (get-in result [:Best 0 :BridgeName]))))
+
+(deftest get-suggested-routes-test
+  (h/test-async :wallet/get-suggested-routes
+    (fn []
+      (let [transaction-type-param  constants/send-type-transfer
+            from-address            "0xAb5801a7D398351b8bE11C439e05C5B3259aeC9B"
+            to-address              "0x53d284357ec70cE289D6D64134DfAc8E511c8a3D"
+            amount-in               "0xA"
+            token-id                "ETH"
+            disabled-from-chain-ids []
+            disabled-to-chain-ids   []
+            network-preferences     []
+            from-locked-amount      {}
+            gas-rates               constants/gas-rate-medium
+            request-params          [transaction-type-param
+                                     from-address
+                                     to-address
+                                     amount-in
+                                     token-id
+                                     disabled-from-chain-ids
+                                     disabled-to-chain-ids
+                                     network-preferences
+                                     gas-rates
+                                     from-locked-amount]]
+        (p/let [result (apply contract-utils/call-rpc "wallet_getSuggestedRoutes" request-params)]
+          (println "RPC Response: " result)
+          (assert-suggested-routes result))))))
+
+;; TODO assertions and add collectibles
+;; currently returning nil
+(deftest get-owned-collectibles-async-test
```

Comment:
Sorry if this might be an irrelevant question : 
As a test to fetch owned collectibles should we give a collectible to this account first? 
OR
does one of the account already has a collectible?

Reply:
Tks for helping!


On the tests, when we generate a new account or even using the recovered one, they contain some assets. Do you we know how this proccess of "give" assets to these test wallets works? 
Because I believe you are right and they dont have any collectibles, and thats why tests like `wallet_getOwnedCollectiblesAsync` returns nil.

@J-Son89 @shivekkhurana Maybe this is Status Go side?

Reply:
This is a real status account so we just need to send some test collectibles to that account as you would with any other account